src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -107,6 +114,11 @@
                     </tbody>
                 </table>
             </div>
+            <div class="form-group submit-group">
+              <button type="submit" class="btn btn-primary btn-save">
+                Check-In selected
```

Comment:
We need to translate it, and I'd put it as ``{% trans "Check-in selected attendees" %}